apps/prairielearn/src/tests/manualGrading.test.js

```diff
@@ -467,9 +467,10 @@ describe('Manual Grading', function () {
         setUser(defaultUser);
         const instancesBody = await (await fetch(instancesAssessmentUrl)).text();
         const $instancesBody = cheerio.load(instancesBody);
-        const token = $instancesBody('form[name=grade-all-form]')
-          .find('input[name=__csrf_token]')
-          .val();
+        const token =
+          $instancesBody('form[name=grade-all-form]')
+            .find('input[name=__csrf_token]')
+            .attr('value') || '';
```

Comment:
This is mostly because, in cheerio 1.0.0, `val()` returns a potential array, while `attr()` always returns a string (or undefined).